src/bundlers/webpack/configs/base.ts

```diff
@@ -88,6 +88,5 @@ export const getBaseConfig = (payloadConfig: SanitizedConfig): Configuration =>
       template: payloadConfig.admin.indexHTML,
       filename: path.normalize('./index.html'),
     }),
-    new webpack.HotModuleReplacementPlugin(),
```

Comment:
I think it would be useful to have this during development. Adding this as part of the dev config https://github.com/payloadcms/payload/blob/master/src/bundlers/webpack/configs/dev.ts

Reply:
During development - yes, and the `dev.ts` config has it. It makes no sense to have it in production and that's essentially what my change accomplishes.

Reply:
Here it is:

https://github.com/payloadcms/payload/blob/0bd335303dff71977b46b373fbee859d11c33337/src/bundlers/webpack/configs/dev.ts#L34-L37

As you can see, `dev.ts` loads the `base.ts` plugins and then the HMR plugin explicitly. There's no need to have it in `base.ts` *as well* because it ends up in the production config.

Reply:
Correct.